backend/tracim_backend/lib/core/content.py

```diff
@@ -1403,6 +1405,10 @@ def move(
         must_stay_in_same_workspace: bool = True,
         new_workspace: Workspace = None,
     ) -> None:
+        if item.content_id == new_parent.content_id:
+            raise ConflictingMoveInItself("You cannot copy a folder into itself")
```

Comment:
Technically, `move` is an operation on any type of content, not specified to folders. For example, even if the user interface does not allow it (at the moment), you can move a comment to another content. So the messages must be more generic:

- "You cannot copy a folder into itself" => "You can't move a content into itself" (by the way, it's not a copy, it's a move)
- "The destination folder is in the source folder" => "You can't move a content into one of its children")